README.md

```diff
@@ -629,6 +633,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Kabir](https://github.com/kabir0x23)
   - [Kabir Jain](https://github.com/kkkkkabir)
   - [Kader Sarikaya](https://github.com/kadersarikaya)
+  - [Kartik suthar](https://github.com/Kartik2220)
```

Comment:
Please rearrange your name according to the alphabetical order

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient